### ISSUE TITLE
EVSYS gen1 not declared?

### ISSUE THREAD

**Issue by FactoryOptic** [SpenceKonde/megaTinyCore#1060]
ATtiny3227

```
#include <Event.h>
#include <Logic.h>

void setup() {
    // EVSYS.ASYNCCH0 = EVSYS_ASYNCCH0_PORTA_PIN6_gc; /* Asynchronous Event from Pin PA6 */
    Event0.set_generator( gen0::pin_pa6 ); // Set pin Pa6 as event generator, Event0

    // EVSYS.ASYNCCH1 = EVSYS_ASYNCCH1_PORTB_PIN6_gc; /* Asynchronous Event from Pin PB6 */
    Event1.set_generator( gen1::pin_pb6 ); // Set pin PB6 as event generator, Event1
}

void loop() {
}
```

Getting:
```
Using library Event at version 1.3.0 in folder: /Users/someone/Library/Arduino15/packages/megaTinyCore/hardware/megaavr/2.6.10/libraries/Event 
Using library Logic at version 1.3.1 in folder: /Users/someone/Library/Arduino15/packages/megaTinyCore/hardware/megaavr/2.6.10/libraries/Logic 
exit status 1

Compilation error: 'gen1' has not been declared
```

**Comment by SpenceKonde** [comment]
Improper tests were carried out to see whether that namespace should exist, and if so, make it available.

**Comment by SpenceKonde** [comment]
Fixed by dc86020
